File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.py
repeat_num = int(input())
N = repeat_num

for _ in range(repeat_num):
    text = input()
    for idx in range(text.__len__() - 1):
        if (text[idx] != text[idx+1]) and (text[idx] in text[idx+2:]):
            N -= 1
            break
print(N)

File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.java
import java.util.*;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int cnt = Integer.parseInt(scanner.nextLine());
        int N = cnt;

        for (int tmp=0; tmp < cnt; tmp++)
        {
            char[] text = scanner.nextLine().toCharArray();
            for (int idx=0; idx < text.length - 1; idx++)
            {
                boolean trig = false;
                if (text[idx] != text[idx+1]) {
                    for (int idx_1=idx+1; idx_1<text.length; idx_1++)
                    {
                        if (text[idx] == text[idx_1])
                        {
                            N -= 1;
                            trig = true;
                            break;
                        }
                    }
                }
                if (trig)
                {
                    break;
                }
            }
        }
        System.out.println(N);
    }
}